python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -211,6 +211,11 @@ def _replace_with_low_bit_linear(model, qtype, modules_to_not_convert=None,
             if (not any(key in ".".join(current_key_name) for key in modules_to_not_convert) and
                     not isinstance(module, LowBitLinear)):
                 in_features, out_features, mp_group = linear_args
+                lm_head = False
+                if name == "lm_head":
+                    from bigdl.llm.transformers.utils import get_optimize_lm_head
+                    if model_type in ["gptj", "llama"] and get_optimize_lm_head():
```

Comment:
only applicable to "gptj" and "llama"?

Reply:
Not all model inputs dim >=3, and seq_len is not always the first dimension. Other models need to be validated before being included.